samples/bluetooth/peripheral_fast_pair/src/main.c

```diff
@@ -306,8 +333,12 @@ void main(void)
 	k_work_init_delayable(&fp_adv_mode_status_led_handle, fp_adv_mode_status_led_handle_fn);
 	k_work_init_delayable(&fp_discoverable_adv_timeout, fp_discoverable_adv_timeout_fn);
 
-	k_work_schedule(&fp_adv_mode_status_led_handle, K_NO_WAIT);
-	k_work_submit(&bt_adv_restart);
+	int ret = k_work_schedule(&fp_adv_mode_status_led_handle, K_NO_WAIT);
```

Comment:
You could use `err` also here, it would simplify a logic a little because then you would not need  `ARG_UNUSED` macro. Also in other places.

Reply:
Discussed offline. The separate value is used to separate value returned here from error codes returned by other functions